crates/transaction-generator-lib/src/publishing/module_simple.rs

```diff
@@ -373,6 +383,37 @@ impl EntryPoints {
                     ],
                 )
             },
+            EntryPoints::ResourceGroupsGlobalResource
+            | EntryPoints::ResourceGroupsIndividualResource => {
+                let rng: &mut StdRng = rng.expect("Must provide RNG");
```

Comment:
If both ResourceGroupsGlobalResource and ResourceGroupsIndividualResource does the same thing, why have 2 different entrypoints?

Reply:
they are not the same.

they call different function (set vs set_p)

set_p is a multisig transaction (i.e. additional signer argument is created automatically based on the signers of the transaction. That allows us to have a different account use their sequence number, and different account to have resources created for.

Reply:
can we have proper names instead of set and set_p? what is p?